src/api.raml

```diff
@@ -837,6 +847,42 @@ traits:
                                 complete:
                                     displayName: Complete
                                     value: !include samples/digest/v1/complete.json
+        put:
```

Comment:
I think we are dropping `POST` (id is always passed in through `PUT`) and `PATCH` (incorrectly implemented, difficult to do) so this verb should suffice